CMakeLists.txt

```diff
@@ -112,9 +112,7 @@ if(PYTHON_BINDINGS)
       set(PYTHON_LIBRARIES "$ENV{PYTHON_LIBRARIES}")
     endif()
     if(NOT PYTHON_INCLUDE_DIRS AND NOT PYTHON_LIBRARIES)
-        find_package(PythonLibs 2.7 REQUIRED)
-      else()
-        message(FATAL_ERROR "Inconsistent PYTHON_INCLUDE_DIRS and PYTHON_LIBRARIES")
```

Comment:
If usr set PYTHON params, the line 117 is always true.  It shall break building.